tardis/base.py

```diff
@@ -16,27 +16,39 @@ def run_tardis(
     specific=None,
 ):
     """
-    This function is one of the core functions to run TARDIS from a given
-    config object.
+    Run TARDIS from a given config object.
 
-    It will return a model object containing
+    It will return a model object containing the TARDIS Simulation.
 
     Parameters
     ----------
     config : str or dict or tardis.io.config_reader.Configuration
         filename of configuration yaml file or dictionary or TARDIS Configuration object
-    atom_data : str or tardis.atomic.AtomData
-        if atom_data is a string it is interpreted as a path to a file storing
+    atom_data : str or tardis.atomic.AtomData, optional
+        If atom_data is a string it is interpreted as a path to a file storing
         the atomic data. Atomic data to use for this TARDIS simulation. If set to None, the
         atomic data will be loaded according to keywords set in the configuration
-        [default=None]
-    virtual_packet_logging : bool
-        option to enable virtual packet logging
-        [default=False]
+    packet_source : class, optional
+        A custom packet source class or a child class of `tardis.montecarlo.packet_source`
+        used to override the TARDIS `BasePacketSource` class.
+    simulation_callbacks : list of lists, optional
+        Set of callbacks to call at the end of every iteration of the Simulation.
+        The format of the lists should look like:
+        [[callback1, callback_arg1], [callback2, callback_arg2], ...]
+        Where the callback function signature should look like:
+        callback_function(simulation, extra_arg1, ...)
+    virtual_packet_logging : bool, default False, optional
```

Comment:
Not sure if you can write default values in parameter type, they should be in parameter description AFAIR

Reply:
I'm afraid the examples mentioned [here](https://numpydoc.readthedocs.io/en/latest/format.html#parameters) have the default values next to parameter type, that's why I used it. 
I didn't follow that convention in #1636 and I will fix it ASAP. `Pandas` also follows the `NumPy` style guide, it seems they did the same too.

Reply:
Oh, I see so numpydoc says both are valid (in type or in description). Thanks for linking up examples

If you're keeping it in type then do it throughout the module - when there are options we should follow one option consistently since that will enhance readability

Reply:
I agree with Jaladh. We shpuld have consistent docstring formats throughour the whole Project.

Reply:
@atharva-2001 I think you are missing this point entirely in your changes :sweat_smile:

Reply:
`default: False` like you did below